setup.py

```diff
@@ -281,9 +283,15 @@ def get_flash_attention3_nvcc_archs_flags(cuda_version: int):
         return []
     if cuda_version < 1203:
         return []
+if ((sys.platform == "win32" or platform.system() == "Windows")
+    and cuda_version >= 1300):
+        return []
```

Comment:
Missing a tab there..?
```suggestion
    if ((sys.platform == "win32" or platform.system() == "Windows")
        and cuda_version >= 1300):
        return []
```